edith-step/src/components/TopNavbar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import Navbar from 'react-bootstrap/Navbar';
+import Nav from 'react-bootstrap/Nav';
+
+/**
+ * Top navigation bar for user interaction with loggin, loggin out
+ * and other app features.
+ */
+class TopNavbar extends React.Component {
+
```

Comment:
Lint error: "Block must not be padded by blank lines"

Please fix.

Reply:
Changes made.